feature/gribi/otg_tests/base_leader_election_test/base_leader_election_test.go

```diff
@@ -61,7 +67,8 @@ var (
 	}
 
 	atePort1 = attrs.Attributes{
-		Name:    "atePort1",
+		Name:    "port1",
+		MAC:     "02:00:01:01:01:01",
```

Comment:
Sorry, I forgot, is the MAC address optional? It would be better if they can be automatically assigned like the ATE API, since most tests don't need to know what MAC addresses to use.

Reply:
The MAC address is mandatory when creating OTG ethernet devices [OTG spec - set config](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml#tag/Configuration/operation/set_config)
Also I don't know why there are no defaults like in the ATE API but I'm asking to see if this could be implemented